internal/database/testing/database.go

```diff
@@ -16,38 +19,53 @@ import (
 // DumpTable dumps the contents of the given table to stdout.
 // This is useful for debugging tests. It is not intended for use
 // in production code.
-func DumpTable(c *gc.C, db *sql.DB, table string) {
-	rows, err := db.Query("SELECT * FROM " + table)
-	c.Assert(err, jc.ErrorIsNil)
-	defer rows.Close()
+func DumpTable(c *gc.C, db *sql.DB, table string, extraTables ...string) {
```

Comment:
I updated this, because I was trying to debug what a test was doing.

It's useful to have multiple tables dumped out.